src/rgw/rgw_admin.cc

```diff
@@ -4996,6 +4996,12 @@ int main(int argc, const char **argv)
     }
     break;
   case OPT_USER_CREATE:
+    if (!store->svc.zone->is_meta_master() && !yes_i_really_mean_it) {
+      cerr << "user created here will not be synced to master zone" << std::endl;
+      cerr << "do you really mean it? (requires --yes-i-really-mean-it)" << std::endl;
```

Comment:
we should also strongly recommend running the command on the master zone instead